AppServer/google/appengine/tools/devappserver2/module.py

```diff
@@ -35,17 +34,17 @@
 import urlparse
 import wsgiref.headers
 
-
 from google.appengine.api import api_base_pb
 from google.appengine.api import apiproxy_stub_map
 from google.appengine.api import appinfo
 from google.appengine.api import request_info
-from google.appengine.api.logservice import logging_capnp
```

Comment:
As discussed, I had run our hawkeye logservice test on this branch and it passed. capnp is still a requirement, but this file seems to only need the stub. I will look into it a bit more though.